src/types/PublicKeyString.ts

```diff
@@ -45,8 +45,9 @@ export function isPublicKeyString(value: string): value is PublicKeyString {
  * @throws {@link AssertionError} if the {@link value} is not a valid base58 string for a PublicKey of a Solana
  * Account.
  * @private
+ * @throws if value is not a valid PublicKey address
  */
-export function tryConvertToPublickKey(value: PublicKeyString): PublicKey {
+export function convertToPublickKey(value: PublicKeyString): PublicKey {
```

Comment:
Maybe not now but it would be nice to include more types here in the future to convert from anything reasonable into a `PublicKey`.

Reply:
Yeah, as outlined in the PR message I:

> put off in order to gain more insight into use cases and ideally find an existing implementation instead of hand rolling our own

Reply:
Sorry I missed that